src/shared-components/button/components/roundButton/style.ts

```diff
@@ -107,10 +107,10 @@ const determineAlternateTextColor = (
  */
 const buttonTextColor = (
   buttonColor: ThemeColors,
-  textColor: string,
   theme: ThemeType,
+  textColor?: string,
```

Comment:
I think we should include constraining this type as part of this PR: 
```suggestion
  textColor?: ThemeType['COLORS'],
```

We would also want to update the JSDoc comment above, since `string` is not a valid `textColor` (and it looks like I didn't update it when I converted `color` to `buttonColor`, as well.)

Reply:
refactored it